src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -0,0 +1,15 @@
+export const description = `
+TODO:
+- test compatibility between bind groups and pipelines
```

Comment:
Should we also test inheritance between pipelines with different layouts for 0 or for the last bindgroup? Also what happens when a pipeline has an empty bindgroup for group 1 but not group 2 and we never set anything to group 1 for example.

Is it possible to unset bindgroups?

Reply:
It's not possible to unset bind groups. If they go unused by a pipeline that's supposed to be OK.

Reply:
> Should we also test inheritance between pipelines with different layouts for 0 or for the last bindgroup? Also what happens when a pipeline has an empty bindgroup for group 1 but not group 2 and we never set anything to group 1 for example.

Added to operation because I think these wouldn't be invalid in validation.